tools/downloader/list_topologies.yml

```diff
@@ -385,6 +385,164 @@ topologies:
     framework: caffe
     license: https://raw.githubusercontent.com/DuinoDu/mtcnn/master/LICENSE
 
+  - name: "mask_rcnn_inception_resnet_v2_atrous_coco"
+    description: >-
+      Mask R-CNN Inception Resnet V2 Atrous trained on COCO dataset. Used for object instance segmentation. For details see paper
+      <https://arxiv.org/pdf/1703.06870.pdf>
+    task_type: "instance segmentation"
+    files:
+      - name: mask_rcnn_inception_resnet_v2_atrous_coco_2018_01_28.tar.gz
+        sha256: 816afc8987c2152cbd0e24af3a6fb82086cea0645a38778884bd166c0d016bb5
+        source: http://download.tensorflow.org/models/object_detection/mask_rcnn_inception_resnet_v2_atrous_coco_2018_01_28.tar.gz
+    output: "instance_segmentation/common/mask_rcnn/mask_rcnn_inception_resnet_v2_atrous_coco/tf"
+    postprocessing:
+      - $type: unpack_archive
+        format: gztar
+        file: mask_rcnn_inception_resnet_v2_atrous_coco_2018_01_28.tar.gz
+    model_optimizer_args:
+      - --framework=tf
+      - --data_type=FP16
```

Comment:
Why do you set for all new instance segmentation models and detection models FP16 data type and for mobilenet FP32? is this done intentionally?